doc/contribute/build_images.rst

```diff
@@ -1,10 +1,39 @@
-CCF Build images
+CCF Build Images
 ================
 
-CCF build images are produced by running the `docker/app_ci <https://github.com/microsoft/CCF/blob/main/docker/ccf_ci>`_ Docker file,
-and pushed to `Docker Hub <https://hub.docker.com/r/ccfciteam/ccf-ci/tags>`_.
+CCF build images are produced by running the :ccf_repo:`docker/ccf_ci` Docker file and pushed to the ``ccfmsrc`` Azure Container Registry. They can be pulled by unauthenticated users:
 
-Pushing a git tag of the form ``ccf_ci_image/$TAG`` will trigger a `workflow <https://github.com/microsoft/CCF/blob/main/.github/workflows/ci-containers.yml>`_
-that builds and pushes a ``ccfciteam/ccf-ci:$TAG`` image.
+.. code-block:: bash
 
-That image can then be used in CI and CD pipelines.
\ No newline at end of file
+    $ docker pull ccfmsrc.azurecr.io/ccf-sgx-ci:<tag>
+
+Pushing a git tag of the form ``ccf_ci_image/$TAG`` will trigger the :ccf_repo:`.github/workflows/ci-containers.yml` workflow that builds and pushes a new ``ccfmsrc.azurecr.io/ccf-sgx-ci:$TAG`` image.
+
+That image can then be used in CI and CD pipelines.
+
+Azure Container Registry Notes
+------------------------------
+
+The ``ccfmsrc.azurecr.io`` Azure Container Registry (ACR) has been setup so that all images can be pulled by unauthenticated users (see `documentation <https://docs.microsoft.com/en-us/azure/container-registry/anonymous-pull-access>`_):
+
+.. code-block:: bash
+    
+    $ az login
+    $ az account set --subscription CCF
+    $ az acr update --name ccfmsrc --anonymous-pull-enabled
+
+The ``ci-push-token`` has been setup so that only authorised users (in this case the :ccf_repo:`.github/workflows/ci-containers.yml` workflow) can push new ``ccf-sgx-ci`` images:
+
+.. code-block:: bash
+
+    $ az login
+    $ az account set --subscription CCF
+    # Create scoped map
+    $ az acr scope-map create --name ci-push --registry ccfmsrc --description "Push CCF CI images"
+    # Add repository, even before repository is created
+    $ az acr scope-map update --name ci-push --registry ccfmsrc --add-repository ccf-sgx-ci content/write content/read
+    # Create token, outputs password to add as GitHub ACR_CI_PUSH_TOKEN_PASSWORD secret
+    $ az acr token create --name ci-push-token --registry ccfmsrc --scope-map ci-push
+
+.. note:: 
+    - The ``ccfmsrc`` ACR instance was upgraded to Premium to enable preview features such as scoped maps.
```

Comment:
What's a scoped map?

Reply:
> A scope map groups the repository permissions you apply to a token, and can reapply to other tokens. Every token is associated with a single scope map.
> With a scope map:
> - Configure multiple tokens with identical permissions to a set of repositories
> - Update token permissions when you add or remove repository actions in the scope map, or apply a different scope map

Source: https://docs.microsoft.com/en-us/azure/container-registry/container-registry-repository-scoped-permissions#concepts

I've added a link to this in the docs.